front/src/containers/Authentication/styleLogin.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components'
```

Comment:
You don't need use 3 styles file, as i commented above. Use a unique "style" file